_posts/2016-11-09-foreman-community-newsletter-january-2016.md

```diff
@@ -0,0 +1,71 @@
+  - [foreman-selinux](https://github.com/theforeman/foreman-selinux) updated to 1.10.0 on 23rd of December 2015
+  - [foreman-installer](https://github.com/theforeman/foreman-installer) updated to 1.10.0 on 23rd of December 2015
+  - [smart-proxy](https://github.com/theforeman/smart-proxy) updated to 1.10.0 on 23rd of December 2015
+  - [smart_proxy_remote_execution_ssh](https://github.com/theforeman/smart_proxy_remote_execution_ssh) updated to 0.0.10 on 22nd of December 2015
+  - [foreman-docker](https://github.com/theforeman/foreman-docker) updated to 2.0.0 on 18th of December 2015
+  - [bastion](https://github.com/Katello/bastion) updated to 2.1.0 on 18.12.2016
+  - [foreman-one](https://github.com/theforeman/foreman-one) updated to 0.4 on 18th of December 2015
+  - [smart_proxy_chef](https://github.com/theforeman/smart_proxy_chef) updated to 0.1.6 on 18th of December 2015
+  - [chef-handler-foreman](https://github.com/theforeman/chef-handler-foreman) updated to 0.1.1 on 18th of December 2015
+  - [hammer_cli_foreman_remote_execution](https://github.com/theforeman/hammer_cli_foreman_remote_execution) updated to 0.0.2 on 17th of December 2015
+  - [foreman_remote_execution](https://github.com/theforeman/foreman_remote_execution) updated to 0.1.2 on 17th of December 2015
+  - [hammer-cli](https://github.com/theforeman/hammer-cli) updated to 0.5.1 on 15th of December
+  - [hammer-cli-foreman](https://github.com/theforeman/hammer-cli-foreman) updated to 0.5.1 on 14th of December
+
+### New media and blogs
+
+- The latest [community demo](https://www.youtube.com/watch?v=gvOFbmZO1yM&list=PLLTIBSsvp9qQwNxhQVtaqNNkMkvsHldGA) took place on December 10th. Highlights:
+  - [Bare metal image-based provisioning (lzap)](https://youtu.be/gvOFbmZO1yM?t=0)
+  - [Remote execution: CLI (stbenjam)](https://youtu.be/gvOFbmZO1yM?t=575)
+  - [Remote execution: Recurrence (aruzicka)](https://youtu.be/gvOFbmZO1yM?t=840)
+  - [Remote execution: Effective user (inecas)](https://youtu.be/gvOFbmZO1yM?t=1220)
+  - [Parameter UI improvements (tbrisker)](https://youtu.be/gvOFbmZO1yM?t=1675)
+  - [Hammer: Setting parameters for Taxonomies (tstrachota)](https://youtu.be/gvOFbmZO1yM?t=2062)
+  - [Hammer: Setting CLI defaults (aglodboi)](https://youtu.be/gvOFbmZO1yM?t=2256)
+  - [Community updates (gwmngilfen)](https://youtu.be/gvOFbmZO1yM?t=2785)
+
+- We held a deep dive about [Taxonomies](https://www.youtube.com/watch?v=D4cON77hmnI) with Daniel Lobato Garcia, covering used Models, how they interact with Roles and Filters and how they interact with other plugins. Thanks Daniel!
+
+### Upcoming events
+- There is another [Deep Dive](https://www.youtube.com/watch?v=0BSnlUkCC7I) planned for Tuesday January 19th, where Dmitri Dolguikh will be talking about the Smart Proxy, aiming to provide an overview of proxy plugins and plugin providers, validators, events and event reporting.
+- [FOSDEM](https://fosdem.org/2016/) takes place on 30th and 31st of January.
```

Comment:
Sure! Please add the following:
In FOSDEM, in the Virt/IAAS devroom, Oved Ourfali will be doing a talk on "Bringing Host Lifecycle and Content Management into oVirt". In this talk Oved will give an overview of oVirt, Foreman, and Katello , and show how we use the latter two in oVirt to give a Powerful Virtualizaed Data-Center Management system. More details are available in:
https://fosdem.org/2016/schedule/event/virt_iaas_host_lifecycle_content_management_in_ovirt/